docker-compose.yml

```diff
@@ -72,6 +77,8 @@ services:
       - vufind
     ports:
       - "8080:8080"
+      - "5173:5173"
```

Comment:
I'm assuming this is so it could run vite/storybook, but that would prob. need to be a separate container